ansible-playbook.yml

```diff
@@ -98,7 +98,7 @@
       become: true
 
     - name: Install linux-headers
-      shell: apt-get install linux-headers-`uname -r`
+      shell: apt-get -y install linux-headers-`uname -r`
```

Comment:
This was already committed as part of another PR -- need to rebase to get rid of this... (Vivian is the queen of git I hear...)